src/cli/cli.py

```diff
@@ -284,3 +299,24 @@ def server_halt(self, cmd: str) -> None:  # pylint: disable=unused-argument
         :param cmd: The rest of the command sent
         """
         self.container_manager.server_halt()
+
+    def ping(self, cmd: str) -> None:  # pylint: disable=unused-argument
+        """
+        Pings the server
+
+        :param cmd: The rest of the command sent
+        """
+        self.container_manager.ping()
+
+    def ssh_address(self, cmd: str) -> None:  # pylint: disable=unused-argument
```

Comment:
This function only works if the server is still running. Is there any way to ensure that we have this information if we lose connection to the server?

Reply:
> Is there any way to ensure that we have this information if we lose connection to the server?
Not really, since the server is what allocates and keeps track of the port.

Reply:
One issue that I have ran into so far a number of times is the server stopping accepting responses. It would be nice if the client had this information so that we could kill the server and the stuff it uses even if it is being a bitch